pyftpdlib/handlers.py

```diff
@@ -3005,7 +3005,7 @@ def ftp_OPTS(self, line):
 
     def ftp_NOOP(self, line):
         """Do nothing."""
-        self.respond("200 I successfully done nothin'.")
+        self.respond("200 I successfully did nothing'.")
```

Comment:
A little bit late, but I've just noticed that now that the abbreviated `nothin'` has been replaced with `nothing`, the apostrophe should be removed ;-)